services/user/CommonApi/common/packages/common-lib/vite.config.ts

```diff
@@ -14,7 +14,8 @@ export default defineConfig({
             name: "@psibase/common-lib",
             fileName: "common-lib",
         },
-        minify: false,
+        minify: true,
+        sourcemap: false,
```

Comment:
@brandonfancher Is this needed; or do we have this sensibly defaulted in the base config?